src/scss/variables.scss

```diff
@@ -0,0 +1,349 @@
+        textMuted: $darkGrey1,
+        backgroundColor: $darkDarkBlue2,
+        badgeDangerBackground: $darkDanger,
+        badgeDangerText: $darkDarkBlue2,
+        badgeInfoBackground: $darkInfo,
+        badgeInfoText: $darkDarkBlue2,
+        badgePrimaryBackground: $darkLinks,
+        badgePrimaryBackgroundHover: $darkPrimary-alt,
+        badgePrimaryText: $darkDarkBlue2,
+        badgeSecondaryBackground: $darkDarkBlue2,
+        badgeSecondaryText: $white,
+        bgPrimaryColor: $darkPrimary,
+        bgLightColor: $darkDarkBlue2,
+        borderColor: $darkBlue1,
+        borderPrimaryColor: $darkPrimary,
+        btnDanger: $darkDanger,
+        btnDangerHover: $darkDangerHover,
+        btnDangerText: $white,
+        btnLinkText: $darkGrey1,
+        btnLinkTextHover: $white,
+        btnOutlineDangerBackground: $darkDanger,
+        btnOutlineDangerBackgroundHover: $darkDangerHover,
+        btnOutlineDangerBorder: $darkDanger,
+        btnOutlineDangerBorderHover: $darkDangerHover,
+        btnOutlineDangerText: $darkDarkBlue2,
+        btnOutlineDangerTextHover: $darkDarkBlue2,
+        btnOutlinePrimaryBackground: $darkPrimary,
+        btnOutlinePrimaryBackgroundHover: $darkPrimary-alt,
+        btnOutlinePrimaryBorder: $darkPrimary,
+        btnOutlinePrimaryBorderHover: $darkPrimary-alt,
+        btnOutlinePrimaryText: $darkDarkBlue2,
+        btnOutlinePrimaryTextHover: $darkDarkBlue2,
+        btnOutlineSecondaryBackground: transparent,
+        btnOutlineSecondaryBackgroundHover: transparent,
+        btnOutlineSecondaryBorder: $darkGrey1,
+        btnOutlineSecondaryBorderHover: $darkGrey2,
+        btnOutlineSecondaryText: $white,
+        btnOutlineSecondaryTextHover: $darkGrey2,
+        btnPrimary: $darkLinks,
+        btnPrimaryBorderHover: $darkPrimary-alt,
+        btnPrimaryHover: $darkPrimary-alt,
+        btnPrimaryText: $darkDarkBlue2,
+        btnSecondary: transparent,
+        btnSecondaryBorder: $darkGrey1,
+        btnSecondaryBorderHover: $darkGrey2,
+        btnSecondaryHover: transparent,
+        btnSecondaryText: $white,
+        btnSecondaryTextHover: $darkGrey2,
+        calloutBackground: $darkBlue2,
+        calloutColor: $white,
+        cdkDraggingBackground: $darkDarkBlue1,
+        codeColor: #E83E8C,
+        dropdownBackground: $darkDarkBlue1,
+        dropdownDangerHover: $darkDangerHover,
+        dropdownHover: rgba(0,0,0,0.03),
+        dropdownTextColor: $white,
+        dropdownTextMuted: #BEC6CF,
+        focus: rgb(106 153 240 / 25%),
+        footerBackgroundColor: $darkBlue1,
+        foregroundColor: $darkDarkBlue1,
+        headerColor: $darkBlue1,
+        iconColor: #777777,
+        iconHover: $darkGrey2,
+        imgFilter: invert(1) grayscale(1),
+        inputBackgroundColor: $darkDarkBlue1,
+        inputBorderColor: $darkGrey1,
+        inputDisabledBackground: $darkBlue2,
+        inputDisabledColor: $darkGrey1,
+        inputPlaceholderColor: $darkGrey1,
```

Comment:
We should use the variables here as well, similar to light theme.

Reply:
Ah my bad. Updated these now